ntoskrnl/ps/process.c

```diff
@@ -586,7 +585,8 @@ PspCreateProcess(OUT PHANDLE ProcessHandle,
                         PROCESS_PRIORITY_NORMAL,
                         Affinity,
                         DirectoryTableBase,
-                        (BOOLEAN)(Process->DefaultHardErrorProcessing & 4));
+                        (BOOLEAN)(Process->DefaultHardErrorProcessing &
```

Comment:
You might want to fix this while you're at it. The BOOLEAN cast does exactly nothing, it needs a `!= 0`, BooleanFlagOn, or some such